support/module/packaged/modules.json

```diff
@@ -15,5 +15,8 @@
 	],
 	"commonjs": [
 		"https://github.com/appcelerator-modules/ti.cloud/releases/download/3.2.11/ti.cloud-commonjs-3.2.11.zip"
+	],
+	"hyperloop": [
+		"https://appcelerator-modules.s3.amazonaws.com/hyperloop-2.2.2.zip"
```

Comment:
Please change to 3.0.0. Eventually, we should create a `hyperloop-releases` repo to store those, which would also have the benefit to attach changelogs and beta-versions.

Reply:
We probably should do that right away. I can't update this link right now because that would break builds since ee don't have a 3.0.0 published yet. That would also trigger the automatic update in SDKs <= 6.3.0 so we shouldn't even publish Hyperloop 3.0.0 the old way?.

Reply:
Maybe we should. Then people with an old SDK can still download it, but that on the other hand still breaks their app because of the bumped api-version that requires 7.0.0 anyway.